deps.edn

```diff
@@ -1,8 +1,8 @@
 {:paths   ["src" "resources" "notebooks"]
  :deps    {org.clojure/clojure {:mvn/version "1.11.1"}}
  :aliases {:dev {:extra-deps {org.clojure/clojure         {:mvn/version "1.11.1"}
-                              techascent/tech.ml.dataset  {:mvn/version "7.011"}
-                              scicloj/tablecloth          {:mvn/version "7.007"
+                              techascent/tech.ml.dataset  {:mvn/version "7.014"}
```

Comment:
worth pointing at release 7.021?

Reply:
Could do, for testing, but as only bringing in TMD for TC, using the TMD corresponding to the TC version used.